README.md

```diff
@@ -104,6 +104,12 @@ the server:
 * To use standard streams(stdin, stdout) of the server process do not set any 
 of the above environment variables and the server will fall back to standard streams. 
 
+* To use a **TCP server**, set the following environment variables before starting the server:
+   * `JDTLS_SERVER_PORT`: the port of the socket to listen to
+   * `JDTLS_SERVER_HOST`: the host name to listen to. If not set, defaults to `localhost`.
```

Comment:
is JDTLS_SERVER_HOST really necessary? This *is* the server so always runs on localhost